src/components/expanded-state/custom-gas/FeesPanel.js

```diff
@@ -443,56 +478,193 @@ export default function FeesPanel({
 
   useEffect(() => {
     // validate not zero
-    if (!maxBaseFee || isZero(maxBaseFee)) {
+    if (!maxBaseFee || isZero(maxBaseFee) || greaterThan(1, maxBaseFee)) {
       setMaxBaseFeeError('1 Gwei to avoid failure');
     } else {
       setMaxBaseFeeError(null);
     }
+    const maxBaseFeeToValidate = IS_TESTING === 'true' ? 100 : currentBaseFee;
     if (
-      greaterThan(multiply(MAX_BASE_FEE_RANGE[0], currentBaseFee), maxBaseFee)
+      greaterThan(
+        multiply(MAX_BASE_FEE_RANGE[0], maxBaseFeeToValidate),
+        maxBaseFee
+      )
     ) {
-      setMaxBaseFeeWarning('Lower than suggested');
+      setMaxBaseFeeWarning(LOWER_THAN_SUGGESTED);
     } else if (
-      greaterThan(maxBaseFee, multiply(MAX_BASE_FEE_RANGE[1], currentBaseFee))
+      greaterThan(
+        maxBaseFee,
+        multiply(MAX_BASE_FEE_RANGE[1], maxBaseFeeToValidate)
+      )
     ) {
-      setMaxBaseFeeWarning('Higher than necessary');
+      setMaxBaseFeeWarning(HIGHER_THAN_NECESSARY);
     } else {
       setMaxBaseFeeWarning(null);
     }
   }, [maxBaseFee, currentBaseFee]);
 
   useEffect(() => {
     // validate not zero
-    if (!maxPriorityFee || isZero(maxPriorityFee)) {
-      setMaxPriorityFeeError('1 Gwei to avoid failure');
+    if (!maxPriorityFee) {
+      setMaxPriorityFeeError('0 Gwei to avoid failure');
```

Comment:
shouldn't be 1 here?

Reply:
ah, you can actually pass 0 as the miner tip

Reply:
hm did christian ask for this change? seems strange to me we would recommend 0 gwei even if it is possible